glyphs-reader/src/font.rs

```diff
@@ -4861,18 +4894,22 @@ etc;
         let font = Font::load(&glyphs3_dir().join("COLRv1-grayscale.glyphs")).unwrap();
         assert_eq!(
             vec![
-                Color {
-                    r: 64,
-                    g: 64,
-                    b: 64,
-                    a: 255,
+                ColorStop {
```

Comment:
Adding constructors would make this much less noisy. I'd probably go with `Color::rgba` over `Color::new` just for the self-documenting aspect, but 🤷